Utilities/Maintenance/SourceTarball.bash

```diff
@@ -36,27 +36,20 @@ return_pipe_status() {
 
 find_data_objects() {
   git ls-tree --full-tree -r "$1" |
-  egrep '\.(md5)$' |
+  egrep '\.(cid)$' |
   while read mode type obj path; do
     case "$path" in
-      *.md5)  echo MD5/$(git cat-file blob $obj) ;;
+      *.cid)  echo CID/$(git cat-file blob $obj) ;;
       *)      die "Unknown ExternalData content link: $path" ;;
     esac
   done | sort | uniq
   return_pipe_status
 }
 
-validate_MD5() {
-  md5sum=$(md5sum "$1" | sed 's/ .*//') &&
-  if test "$md5sum" != "$2"; then
-    die "Object MD5/$2 is corrupt: $1"
-  fi
-}
-
 download_object() {
   algo="$1" ; hash="$2" ; path="$3"
   mkdir -p $(dirname "$path") &&
-  if curl -L "https://www.itk.org/files/ExternalData/$algo/$hash" -o "$path.tmp$$" 1>&2; then
+  if curl -L "http://127.0.01:8080/ipfs/$hash" -o "$path.tmp$$" 1>&2; then
```

Comment:
Double checking: Is this local url correct?
How are we setting up that local url? is that address and port the default values of kubo?

Reply:
Correct, that is the default value for `kubo` -- it can be started by running [ipfs-desktop](https://github.com/ipfs/ipfs-desktop/releases).